flake.nix

```diff
@@ -0,0 +1,98 @@
+{
+  description = "Souffle Datalog bindings for Haskell";
+  inputs = {
+    np.url = "github:nixos/nixpkgs?ref=haskell-updates";
```

Comment:
Can we pin this to a commit? If we always fetch from an upstream channel, it's not deterministic over time.

Reply:
You could take a look at current commit of haskell-updates branch and put that here maybe?

Reply:
This is actually pinned (see `flake.lock`)

Reply:
Ah yes I see it now, my bad!